src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorDiagnosticConverter.cs

```diff
@@ -41,6 +41,7 @@ internal static DiagnosticSeverity ConvertSeverity(RazorDiagnosticSeverity sever
             return severity switch
             {
                 RazorDiagnosticSeverity.Error => DiagnosticSeverity.Error,
+                RazorDiagnosticSeverity.Warning => DiagnosticSeverity.Warning,
```

Comment:
Was this just a bit of completeness you noticed or is there a specific reason we need this in this PR?

Reply:
This change exists in main-vs-deps, but wasn't in main. It's kinda how I noticed this issue.